src/js/controller/instream-adapter.js

```diff
@@ -174,7 +174,7 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
 
     function _loadNextItem() {
         _arrayIndex++;
-        _this.loadItem(_array);
+        _this.loadItem(_array).catch(function() { });
```

Comment:
Does it make sense to use `noop` here or no since we're only using it this one time in this file?

Reply:
Would be nice to use `noop` since there is another place where we can use it: https://github.com/jwplayer/jwplayer/pull/3423/files#diff-79d1ee28e72759418dcb9c1e8855dd7cL56

Reply:
this is a no-op. There is a lot of overhead in importing modules, so no real gain in importing `utils.noop`.

Reply:
I was meaning having 
`
const noop = function() { };
`
to use it within the file